packages/esm-esbuild/esbuild.js

```diff
@@ -0,0 +1,21 @@
+import { build } from "esbuild";
+
+build({
+  entryPoints: ["./src/index.ts"],
+  outdir: "./dist",
+  bundle: true,
+  // platform: "node",
+  format: "esm",
+  // target: "node16",
+  plugins: [
```

Comment:
I patch it by force esbuild to use CommonJS version instead of ESM:

```javascript
const forceCommonJS = (modulePath) => ({
  name: `force-common-js-${modulePath}`,
  setup(build) {
    build.onResolve({ filter: /.*/ }, (args) => {
      if (args.path === modulePath) {
        return {
          path: require.resolve(modulePath),
        }
      }
    })
  },
})

//  ...
build({
  // ...
  plugins: [
    forceCommonJS("mappersmith")
  ]
})
```